src/App/App.js

```diff
@@ -10,40 +10,73 @@ import Header from '../Header/Header.js';
 import Login from '../Login/Login.js'
 
 // import React, { Component } from 'react';
-import { BrowserRouter, Route, Switch, withRouter } from 'react-router-dom';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+
 //
 //
 
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      currentUser: '',
+      userId: 0,
+      isLoggedIn: false
+    }
+  }
+
+  setCurrentUser = (data) => {
+    this.setState({ currentUser: data.user.name, userID: data.user.id, isLoggedIn: true})
+  }
+
+  render = () => {
+    return (
+      <main>
+        <BrowserRouter>
+        <Route path='/login' >
+          <Login setCurrentUser={this.setCurrentUser} />
+        </Route>
+        <Route path='/movie' >
+          <MovieData />
+        </Route>
+        </BrowserRouter>
+      </main>
+    )
+  }
+}
+
 // function App() {
 //   return (
-//     <div>
-//       <switch>
-//       <Login />
-//       </switch>
-//       <Header />
-//       <MovieData />
-//     </div>
-//   )
+//     <BrowserRouter>
```

Comment:
Same thing - just delete code instead of commenting it out :) Believe in yourself! Trust that you can rewrite it!